src/logic/calculate.js

```diff
@@ -0,0 +1,23 @@
+const calculate = (dataObject, simbol) => {
+  const { total } = dataObject;
+  const { next } = dataObject;
+  let { operation } = dataObject;
```

Comment:
- [ ] In here, you can simply destructure the object in its properties in one line, they can all be constant because you are not changing the object you are receiving:
`const { total, next, operation } = dataObject`